controller/space.go

```diff
@@ -339,6 +340,11 @@ func ConvertSpace(request *goa.RequestData, p *space.Space, additional ...SpaceC
 					Related: &relatedAreaList,
 				},
 			},
+			Backlog: &app.RelationGeneric{
```

Comment:
I think this should be in link: not relationships. It's not intended to use the backlog to manipulate the relation between Space and WorkItems.